examples/boltzmann_wealth_model_network/README.md

```diff
@@ -0,0 +1,47 @@
+# Boltzmann Wealth Model with Network
+
+## Summary
+
+This is the same Boltzmann Wealth Model, but with a network grid implementation.
+
+A simple model of agents exchanging wealth. All agents start with the same amount of money. Every step, each agent with one unit of money or more gives one unit of wealth to another random agent. This is the model described in the [Intro Tutorial](http://mesa.readthedocs.io/en/latest/intro-tutorial.html).
+
+As the model runs, the distribution of wealth among agents goes from being perfectly uniform (all agents have the same starting wealth), to highly skewed -- a small number have high wealth, more have none at all.
+
+## Installation
+
+To install the dependencies use pip and the requirements.txt in this directory. e.g.
+
+```
+    $ pip install -r requirements.txt
+```
+
+## How to Run
+
+To run the model interactively, run ``run.py`` in this directory. e.g.
+
+```
+    $ python run.py
+```
+
+Then open your browser to [http://127.0.0.1:8521/](http://127.0.0.1:8521/) and press Reset, then Run.
+
+## Files
+
+* ``run.py``: Launches a model visualization server.
+* ``model.py``: Contains the agent class, and the overall model class.
+* ``NetworkGrid.py``: Contains the grid class with a network topology.
+* ``server.py``: Defines classes for visualizing the model in the browser via Mesa's modular server, and instantiates a visualization server.
+* ``NetworkVisualization.py``: Contains the NetworkElement class, necessary to the visualization of the network, using the [sigma.js](http://sigmajs.org/) framework.
```

Comment:
No need to mention this, because this file doesn't exist in the model. You can delete this line.